karma.conf.js

```diff
@@ -13,7 +13,10 @@ module.exports = function (config) {
       require('@angular-devkit/build-angular/plugins/karma')
     ],
     client:{
-      clearContext: false // leave Jasmine Spec Runner output visible in browser
+      clearContext: false, // leave Jasmine Spec Runner output visible in browser
+      jasmine:{
+        random: false
```

Comment:
Why disabling random?

Reply:
To run the test cases in order without random seed

Reply:
Did you fall on errors due to random execution?

Reply:
No i did not..i have removed now in latest commit.